src/ripple/rpc/handlers/AMMInfo.cpp

```diff
@@ -78,32 +78,54 @@ doAMMInfo(RPC::JsonContext& context)
 
     Issue issue1;
```

Comment:
`Issue` probably shouldn't have a default constructor at all. At any rate, consider making this a `std::optional` until it has a valid value.